.changeset/shaggy-tools-trade.md

```diff
@@ -0,0 +1,13 @@
+---
+'braid-design-system': patch
+---
+
+---
+
+updated:
+
+- TextDropdown
+
```

Comment:
```suggestion
updated:
  - TextDropdown
```

![helping](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.imgflip.com%2F1pk2po.jpg&f=1&nofb=1&ipt=205e1a75d5bbbe90a73593494ae0029c3a45e1fd2220186aa3505566f811f018&ipo=images)

Reply:
Prettier formats it like this, I think because it doesn't understand the double frontmatter.